src/app/src/components/DashboardAdjustMatchCard.jsx

```diff
@@ -185,6 +193,8 @@ export default function DashboardAdjustMatchCard({
             );
         }
 
+        const invalidStatus = !isCompleteStatus(match.status);
```

Comment:
"Invalid" makes me think the status value was not in the enum. Consider `matchIsNotComplete`.

Reply:
Updated